src/components/site-header/AvatarDropdown.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { NavLink } from 'react-router-dom';
+import { getConfig } from '@edx/frontend-platform/config';
+import { AppContext } from '@edx/frontend-platform/react';
+import { AvatarButton, Dropdown } from '@edx/paragon';
+
+const AvatarDropdown = ({ showLabel }) => {
+  const { BASE_URL, LMS_BASE_URL, LOGOUT_URL } = getConfig();
+  const { enterpriseConfig, authenticatedUser } = useContext(AppContext);
+  const { username, profileImage } = authenticatedUser;
+  const dashboardLink = `/${enterpriseConfig.slug}`;
```

Comment:
tiniest of nits: maybe `enterpriseDashboardLink` since we do reference the lms dashboard in this file?